### PR TITLE
Lmaoo169 3

### PR DESCRIPTION
**Pull request by LewisWelch94**

### CODE REVIEW

File: Css/editUserModal.css [hunk 1]
```diff
@@ -5,4 +5,9 @@
 
 #uploadImagelabel {
     margin-top: 10px;
+}
+
+#uploadImageText {
+    margin-left: 175px;
```
Comment (OnikNoor97) [inline Css/editUserModal.css:11]: Remove the Margin-left, better to leave it in the left side

File: Global/editUserModal.php [hunk 2]
```diff
@@ -24,6 +23,9 @@
                         <label class="custom-file-label" for="uploadImage">Choose file</label>
                     </div>
                 </div>
+                <div id="uploadImageText" hidden>
+                    <small>Invalid file type.</small>
```
Comment (OnikNoor97) [inline Global/editUserModal.php:27]: Keep it hidden as default

Comment (OnikNoor97) [reply Global/editUserModal.php:27]: Use is-invalid class (Bootstrap) instead of using CSS red

File: Css/editUserModal.css (hunk 1 above)
Comment (LewisWelch94) [reply Css/editUserModal.css]: Done

File: Global/editUserModal.php (hunk 2 above)
Comment (LewisWelch94) [reply Global/editUserModal.php:27]: Done

**Pull request merged**